Lesson06/scripts/hw14.js

```diff
@@ -0,0 +1,20 @@
+console.log("Hello, World!");
+
+//Реалізуйте функцію generateKey(length, characters), яка повертає рядок 
+//випадкових символів із набору characters довжиною length. span>
+
+const characters = 'abcdefghijklmnopqrstuvwxyz0123456789';
+const key = generateKey(16, characters);
+console.log(key); 
+
+function generateKey(lengthRandomArrey, string){
+   const randomArrey=[];
+   const stringToArrey = string.split('');
+   
+   for(let i = 0; i < lengthRandomArrey ; i++ ) {
+
+    randomArrey.push(stringToArrey[Math.floor(Math.random() * stringToArrey.length)]);
```

Comment:
```suggestion
    for(let i = 0; i < lengthRandomArrey ; i++ ) {

    randomArrey.push(string[Math.floor(Math.random() * string.length)]);
```

не обовʼязково перетворювати рядок на маси щоб дістатися до конкретного символа